docs/userguide/dockernetworks.md

```diff
@@ -0,0 +1,511 @@
+<!--[metadata]>
++++
+title = "Docker container networking"
+description = "How do we connect docker containers within and across hosts ?"
+keywords = ["Examples, Usage, network, docker, documentation, user guide, multihost, cluster"]
+[menu.main]
+parent = "smn_containers"
+weight = 3
++++
+<![end-metadata]-->
+
+# Docker container networking
+
+So far we've been introduced to some [basic Docker
+concepts](/userguide/usingdocker/), seen how to work with [Docker
+images](/userguide/dockerimages/) as well as learned about basic [networking
+and links between containers](/userguide/dockerlinks/). In this section
+we're going to discuss how you can take control over more advanced 
+container networking.
+
+This section makes use of `docker network` commands and outputs to explain the
+advanced networking functionality supported by Docker.
+
+# Default Networks
+
+By default, docker creates 3 networks using 3 different network drivers :
+
+```
+$ sudo docker network ls
+NETWORK ID          NAME                DRIVER
+7fca4eb8c647        bridge              bridge
+9f904ee27bf5        none                null
```

Comment:
okay. thats a backend change, and the driver must register itself as none and it will be done :).
Not in this PR though.